README.md

```diff
@@ -116,6 +120,75 @@ This information has as its origin the 46nd minute, of the 2nd hour of February,
 
 [Top](#section0)
 
+###<a id="section1.2"></a> Updating aggregated time series information
+
+As already mentioned, there are 2 main ways to update the aggregated time series information associated to attributes.
+The so-called minimalist option and the formal one.
+
+Regarding the formal option (based on using the Cygnus component for the updating), please refer to the documentation available at the
+<a href="https://github.com/telefonicaid/fiware-cygnus" target="_blank">Cygnus component repository</a>, and more concretely at the following links:
+
+* <a href="https://github.com/telefonicaid/fiware-cygnus/tree/master/flume" href="_blank">Cygnus connector documentation</a>
+* <a href="https://github.com/telefonicaid/fiware-cygnus/tree/master/flume#orion-subscription" href="_blank">Orion subscription</a>
+
+The another option to update the aggregated time series information consists on directly subscribing the STH component
+to the Orion Context Broker to receive the corresponding notifications and generate and update the aggregated data.
+
+To subscribe the STH component to the Orion Context Broker as a way to receive attribute value updates, the following
+curl command can be used:
+
+<pre>
+curl orion.contextBroker.host:1026/v1/subscribeContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- &lt;&lt;EOF
+{
+    "entities": [
+        {
+            "type": "Room",
+            "isPattern": "false",
+            "id": "Room1-gtv"
+        }
+    ],
+    "attributes": [
+        "temperature"
+    ],
+    "reference": "http://sth.host:1028/STH/v1/notify",
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/58fa15d352bf6c8aac640ec5d680c2ca92115060